imgui.go

```diff
@@ -638,3 +638,21 @@ func IsItemHoveredV(flags int) bool {
 func IsItemHovered() bool {
 	return IsItemHoveredV(HoveredFlagsDefault)
 }
+
+// ListBox creates a imgui::Listbox
+func ListBox(label string, currentItem *int32, items []string, itemsCount int, heightItems int) bool {
```

Comment:
In the case of Go, itemsCount would be redundant, as len(items) would be more idiomatic here. This parameter can be removed.